buildpipeline/pipeline.json

```diff
@@ -45,6 +45,21 @@
           }
         },
         {
+          "Name": "Core-Setup-Linux-BT",
+          "Parameters": {
+            "PB_DistroRid": "alpine.3.6-x64",
+            "PB_DockerTag": "alpine-3.6-3148f11-20171119021156",
+            "PB_TargetArchitecture": "x64",
+            "PB_AdditionalBuildArguments":"-TargetArchitecture=x64 -DistroRid=alpine.3.6-x64 -PortableBuild=false -strip-symbols",
```

Comment:
`-DistroRid=alpine.3.6-x64`

We no longer use `DistroRid` on core-setup.  Most distros will calculate the Rid if we have the PlatformAbstractions updated correctly.  So it should just calculate the correct RID based on the machine.

For the cases where we can't calculate the RID correctly, use `/p:OutputRid=<rid>`.  We use that for the `tizen` RID.